backend/utils/tests/test_migrate_with_lock.py

```
@@ -0,0 +1,72 @@
+        self.ensure_connection_calls += 1
+
+    def cursor(self):
+        return FakeCursor(self)
+
+
+class MigrateWithLockCommandTests(SimpleTestCase):
+    def run_command(self, connection):
+        stdout = StringIO()
+        command = Command(stdout=stdout, stderr=StringIO())
+        with patch.object(migrate_with_lock, 'connections', {'default': connection}):
+            with patch.object(migrate_with_lock, 'call_command') as call_command:
```

Ruff (0.15.20)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)